posthog/queries/trends/formula.py

```diff
@@ -26,11 +26,14 @@ def _run_formula_query(self, filter: Filter, team: Team):
             queries.append(sql)
             params = {**params, **entity_params}
 
-        breakdown_value = (
-            ", sub_A.breakdown_value"
-            if filter.breakdown_type == "cohort"
-            else f", {trim_quotes_expr('sub_A.breakdown_value')}"
-        )
+        breakdown_value = ""
+        if filter.breakdown_type == "cohort":
+            breakdown_values_list = ", ".join(f"sub_{letter}.breakdown_value" for letter in letters)
+            breakdown_value = ", arrayFilter(x -> x != 0, [{}])[1]".format(breakdown_values_list)
+        else:
+            breakdown_values_list = ", ".join(trim_quotes_expr(f"sub_{letter}.breakdown_value") for letter in letters)
+            breakdown_value = ", arrayFilter(x -> notEmpty(x), [{}])[1]".format(breakdown_values_list)
```

Comment:
What happens when the empty string is the actual breakdown value of a series?

Reply:
If all are empty, the array will have nothing, and accessing `[1]` will then give us an empty value

Reply:
Can you add a test for that too so that we make sure this never breaks?